joplin/css/layouts/headers.scss

```diff
@@ -21,10 +21,10 @@
     width: calc(100% - #{$nav-wrapper-width});
     background: $coa-color-gray;
     position: fixed;
+    height: 80px;
```

Comment:
copying from a prior PR with just this:

based on this note: https://github.com/cityofaustin/techstack/issues/2827#issuecomment-537265221

"When page title breaks to two lines, the tab row still displays as expected"

This just used a fixed height for the header to avoid odd rearrangement issues. This should not be considered a permanent fix